partition/roles/metal-core/files/bgp-neighbor-stats.service

```diff
@@ -0,0 +1,6 @@
+[Unit]
+Description=Docker System Prune
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/bgp-neighbor-stats.sh
```

Comment:
Is it necessary to run as root user?

Reply:
Yes, because it must be compatible with cumulus and sonic, where we have two different main users, (cumulus and admin) and i dont want to create a new user for this purpose.

Reply:
Good point.